_pages/production.md

```diff
@@ -2,81 +2,150 @@
 lead: >
   Once you’ve tested your integration in our sandbox environment, you can request deployment to the Login.gov production environment.
+  
 redirect_from:
   - /production-deployment/
 sidenav:
   - text: Production endpoints
     href: "#production-endpoints"
-  - text: Deployments
-    href: "#deployments"
-  - text: Confirm IAA
-    href: "#confirm-iaa"
-  - text: Staging environment
-    href: "#staging-environment"
-  - text: Production configuration
-    href: "#production-configuration"
+  - text: Before deployment
+    href: "#before-deployment"
+  - text: Production configuration process
+    href: "#production-configuration-process"
+  - text: Production endpoints
+    href: "#production-endpoints"
+  - text: Confirm Interagency Agreement (IAA)
+    href: "#confirm-interagency-agreement-iaa"
   - text: Request deployment
     href: "#request-deployment"
+  - text: Staging environment
+    href: "#staging-environment"
   - text: Changes to production applications
     href: "#changes-to-production-applications"
 saml_year: 2022
 ---
+**Please note: The deployment process will take 1 to 2 weeks to complete**
 
-## Production endpoints
+Before deployment
+-----------------
 
-Our integration documentation includes endpoint urls for our sandbox environment <https://idp.int.identitysandbox.gov/>. Our production environment is located at <https://secure.login.gov/>. The URL path to each endpoint remains the same. Only the domain will change. For example, the authorization endpoint will change as follows:
-* OpenID Connect: <https://secure.login.gov/openid_connect/authorize>
-* SAML: <https://secure.login.gov/api/saml/auth{{ page.saml_year }}>
+Make sure you have the following items ready before you start the deployment process:
 
-Please be aware that the IDP certificate (X509 Certificate) in the production environment is different from the IDP certificate in the sandbox environment. The production IDP certificates can be found here:
-* OpenID Connect: <https://secure.login.gov/api/openid_connect/certs>
-* SAML: <https://secure.login.gov/api/saml/metadata{{ page.saml_year }}>
+-   [Signed Interagency Agreement (IAA) listing this integration ](https://docs.google.com/document/d/1JrseKerp-FF_uiGElew9z-ltT_l68dgmSFU7so8tQjE/edit#heading=h.gxf7gmdb96uf)
 
-## Deployments
+-   Authority To Operate (ATO) approved environment
 
-All changes to integrations between Login.gov and your application must be reviewed and deployed. **We ask for at least 2 weeks notice for new integrations and changes to existing integrations. Push Notification URLs may require 3 weeks notice in order to allow the domain for outbound communication.** Regular deployments occur every Thursday by the close of the business day. If the regular deployment occurs on a holiday, then it will be completed the following Monday.
+-   [New app on the dashboard ](https://docs.google.com/document/d/1JrseKerp-FF_uiGElew9z-ltT_l68dgmSFU7so8tQjE/edit#heading=h.rl9bxiu1u854)
```

Comment:
1. there's an extra space here in this link
2. this links to a private (to us) doc, is the goal to link to a different heading in this same article?
3. 
```suggestion
-   [New app on the dashboard](https://docs.google.com/document/d/1JrseKerp-FF_uiGElew9z-ltT_l68dgmSFU7so8tQjE/edit#heading=h.rl9bxiu1u854)
```